recipe/meta.yaml

```diff
@@ -48,11 +55,11 @@ requirements:
 test:
   commands:
     - $R -e "library('arrow')"           # [not win]
-    - "\"%R%\" -e \"library('arrow')\""  # [win]
+    - "\"%R%\" -e \"library('arrow'); data(mtcars); write_parquet(mtcars, 'test.parquet')\""  # [win]
```

Comment:
Why not run the full test suite?

Reply:
We normally only run very simple (import) tests in conda-forge as test suits are often very expensive. In this case, I should probably run it as we doing experimental stuff.